### PR TITLE
Add classes to html tags for easy selection

### PR DESCRIPTION
**Pull request by ragurney**

### CODE REVIEW

**Comment by kkajla12** [comment]
👍

File: app/views/images/_image_card.html.erb [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+  <%= link_to image_tag(image.url, class: 'card-img image-tile img-circle'),
+              image_path(id: image.id),
+              class: 'link_to_image'%>
   <div class = "card-block">
-    <% image.tags.each do |tag| %>
-        <%= link_to images_path(tag: tag.name) do%>
-          <span class="tag tag-default image-tag"><%= tag.name %></span>
-        <% end %>
-    <% end %>
+    <div class="js-image-tags">
```
Comment (rmacklin) [inline app/views/images/_image_card.html.erb]: This div is unnecessary as the `js-image-tags` class can be added on the parent div

File: app/views/images/_image_card.html.erb [hunk 2]
```diff
@@ -1,11 +1,15 @@
-<div class="text-xs-center card" data-image-id="<%= image.id %>">
-  <%= image_tag(image.url, class: 'card-img image-tile img-circle') %>
+<div class="js-image-card-container text-xs-center card" data-image-id="<%= image.id %>">
+  <%= link_to image_tag(image.url, class: 'card-img image-tile img-circle'),
+              image_path(id: image.id),
+              class: 'link_to_image'%>
```
Comment (rmacklin) [inline app/views/images/_image_card.html.erb]: This class name should be something like `image-link`. Classes shouldn't use underscores as the word separator

File: app/views/images/_image_card.html.erb [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
   <div class = "card-block">
```
Comment (rmacklin) [inline app/views/images/_image_card.html.erb]: Avoid putting spaces around the `=` between html attribute names and values.

File: app/views/images/_image_card.html.erb [hunk 4]
```diff
@@ -1,11 +1,15 @@
-<div class="text-xs-center card" data-image-id="<%= image.id %>">
-  <%= image_tag(image.url, class: 'card-img image-tile img-circle') %>
+<div class="js-image-card-container text-xs-center card" data-image-id="<%= image.id %>">
+  <%= link_to image_tag(image.url, class: 'card-img image-tile img-circle'),
```
Comment (rmacklin) [inline app/views/images/_image_card.html.erb]: Adding this link should have been its own atomic commit

**Pull request merged**